subsys/zigbee/Kconfig

```diff
@@ -201,6 +201,62 @@ config ZBOSS_TRAF_DUMP
 	help
 	  Dumps all packets destined to the node over ZBOSS binary trace protocol
 
+choice
+	prompt "Select ZBOSS trace logging method: hexdump, binary, NCP transport"
+	default ZBOSS_TRACE_HEXDUMP_LOGGING
+
+config ZBOSS_TRACE_HEXDUMP_LOGGING
+	bool "Enable logging hexdumped ZBOSS Traces"
+	select RING_BUFFER
+
+config ZBOSS_TRACE_BINARY_LOGGING
+	bool "Enable logging binary ZBOSS Traces"
+	select SERIAL
+	select RING_BUFFER
```

Comment:
Just a side comment. In Zephyr upstream, there is a different approach. Such configs have `depends on` as `select` might cause some side effects. If this is the common way how you are doing it I am fine with that.